lib/pure/unicode.nim

```diff
@@ -25,7 +30,12 @@ proc `==`*(a, b: Rune): bool = return int(a) == int(b)
 template ones(n: untyped): untyped = ((1 shl n)-1)
 
 proc runeLen*(s: string): int {.rtl, extern: "nuc$1".} =
-  ## Returns the number of Unicode characters of the string ``s``
+  ## Returns the number of Unicode characters of the string ``s``.
```

Comment:
IMO we should completely remove the term "Unicode character" from the docs, it's misleading. Instead, `Rune` should be documented as a "Unicode code point" and the rest of the docs should use the term "rune" (so this doc comment would be "Returns the number of runes of the string ``s``".

Reply:
Resolved in the new commit.